test/model.test.js

```diff
@@ -8115,4 +8115,12 @@ function pick(obj, keys) {
     }
   }
   return newObj;
-}
\ No newline at end of file
+}
+
+describe('Check if statics functions that is supplied in schema option is availabe', function() {
+  it('should give an array back rather than undefined m0_0aAutoTyped', function M0_0aModelJS() {
+    const testSchema = new mongoose.Schema({}, { statics: { staticFn() { return 'Returned from staticFn'; } } });
```

Comment:
so do we add a new option?
Probably also means that we have to modify the docs too.

@vkarpov15 
Is adding statics here ok? Or does it could it break something with loadClass or so?